ui/admin/app/styles/app.scss

```diff
@@ -735,6 +735,17 @@
   /* stylelint-enable selector-class-pattern */
 }
 
+.config-tags {
+  /* stylelint-disable selector-class-pattern */
+  .hds-table__tbody {
+    .hds-table__td:last-child {
+      // remove width and white-space styles
```

Comment:
Is this removing white space styles?

Reply:
@ZedLi no just the width, changing comment